### PR TITLE
add set_rundir in aim to use the resources folder

### PR DESCRIPTION
**Pull request by PoloNX**
I have updated the xmake.lua. Idk if it breaks something but without this I can't launch the demo.

Updated the readme too

### CODE REVIEW

**Comment by xfangfang** [comment]
cc @zeromake

File: README.md [hunk 1]
```diff
@@ -46,6 +46,14 @@ cmake -B build_pc -DPLATFORM_DESKTOP=ON
 make -C build_pc -j$(nproc)
 ```
 
+If you want to use xmake you have to follow these steps:
+
+```bash
+xrepo add-repo zeromake-repo https://github.com/zeromake/xrepo
+xmake
+xmake run demo
```
Comment (zeromake) [inline README.md:54]: example option is borealis self use xmake repo import skip demo build
```sh
xrepo add-repo zeromake-repo https://github.com/zeromake/xrepo
xmake f -c --example=y
xmake build demo
xmake run demo
```

File: xmake.lua [hunk 2]
```diff
@@ -132,28 +132,29 @@ target("borealis")
...
-            end)
-        end
-        if is_plat("mingw") then
-            add_ldflags("-static")
-        end
-        if is_plat("windows", "mingw") then
-            add_files("demo/resource.rc")
-        end
-end
+
```
Comment (zeromake) [inline xmake.lua:135]: example option is borealis self use xmake repo import skip demo build, so no modifications are required

File: xmake.lua [hunk 3]
```diff
@@ -132,28 +132,29 @@ target("borealis")
...
+            import("uwp")(target)
+        end)
+    end
+    if is_plat("mingw") then
+        add_ldflags("-static")
+    end
+    if is_plat("windows", "mingw") then
+        add_files("demo/resource.rc")
+    end
+    set_rundir("$(projectdir)")
```
Comment (zeromake) [inline xmake.lua:159]: `set_rundir` is useful

**Comment by dragonflylee** [comment]
> I have updated the xmake.lua. Idk if it breaks something but without this I can't launch the demo.
> 
> Updated the readme too

run follow command to enable demo target

```powershell
xmake f -y --example=y 
```

**Comment by zeromake** [comment]
https://github.com/xfangfang/borealis/commit/b57889ac69830482d4a6fe83bb1c05427d418edb

**Pull request closed without merging**